polaris-core/src/main/java/io/polaris/core/auth/PolarisAuthorizableOperation.java

```diff
@@ -187,6 +187,7 @@ public enum PolarisAuthorizableOperation {
   REVOKE_VIEW_GRANT_FROM_CATALOG_ROLE(
       VIEW_MANAGE_GRANTS_ON_SECURABLE, CATALOG_ROLE_MANAGE_GRANTS_FOR_GRANTEE),
   LIST_GRANTS_ON_VIEW(VIEW_LIST_GRANTS),
+  CREATE_TABLE_DIRECT_WITH_WRITE_DELEGATION(EnumSet.of(TABLE_CREATE, TABLE_WRITE_DATA)),
```

Comment:
Minor nit: Since this enum doesn't need to be sorted by any integer codes reflecting the order of introduction, this op might make the most sense between line 91 and 92 grouped with the other `CREATE_TABLE_` ops.